CMakeLists.txt

```
@@ -978,17 +979,19 @@ if(SDLIMAGE_WEBP)
         target_include_directories(${sdl3_image_target_name} PRIVATE external/libwebp/src)
         add_library(WebP::webp ALIAS webp)
         add_library(WebP::webpdemux ALIAS webpdemux)
+        add_library(WebP::webpmux ALIAS webpmux)
```

I think it has a [`lib` prefix](https://github.com/libsdl-org/libwebp/blob/fa992316e4517b25a4e667a3668b930170becd56/CMakeLists.txt#L528C3-L528C14).
```suggestion
        add_library(WebP::webpmux ALIAS libwebpmux)
```

---

> Its brothers above don't ??

Tell that to the Google devs ;)
The source never lies!!

---

@slouken
There are multiple occurrences of `webpmux`  that needs to renamed to `libwebpmux`.

---

Can you check and make sure I changed the right ones?